src/tests/system/tests/test_netgroups.py

```diff
@@ -241,3 +243,126 @@ def test_netgroup__lookup_nested_groups_with_host_and_domain_values_present(
     result = client.tools.getent.netgroup("nested_group")
     assert result is not None
     assert expected in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(bz=802207)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__fully_qualified_names(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD should not hang when fully qualified names set to true
+    :setup:
+        1. Configure SSSD domain to use fully qualified names
+        2. Create user "user-1"
+        3. A netgroup named "ng-1" is created, "user-1" is added to this netgroup
+    :steps:
+        1. Loop runs 3 iterations to verify the existence and membership of the netgroup "ng-1"
+    :expectedresults:
+        1. In every iteration SSSD should return netgroup "ng-1" and members of the netgroup
+    :customerscenario: True
+    """
+    client.sssd.dom("test")["use_fully_qualified_names"] = "true"
+    user = provider.user("user-1").add()
+    provider.netgroup("ng-1").add().add_member(user=user)
+    client.sssd.start()
+
+    for _ in range(3):
+        result = client.tools.getent.netgroup("ng-1")
+        assert result is not None and result.name == "ng-1", "'ng-1' Netgroup name does not matched!"
+        assert len(result.members) == 1, "'ng-1' contains more than 1 member!"
+        assert "(-, user-1)" in result.members, "'ng-1' members did not match the expected ones!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(bz=678410)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__entry_negative_timeout(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD should not show recently deleted users after entry_negative_timeout
+    :setup:
+        1. Create user with password
+    :steps:
+        1. Check user is present and can be used for authentication
+        2. Delete the user
+        3. Ensure user exists in SSSD cache immediately after deletion
+        4. Try ssh authentication for deleted user
+        5. Check after default entry_negative_timeout user is no longer present in SSSD cache
+    :expectedresults:
+        1. User can be used for authentication
+        2. User is deleted
+        3. User exists in SSSD cache immediately after deletion
+        4. Fail ssh authentication for deleted user
+        5. After default entry_negative_timeout user is no longer present in SSSD cache
+    :customerscenario: True
+    """
```

Comment:
Where are You setting entry_negative_timeout?

Reply:
Its default one

Reply:
How about setting it to something lower. That way:
1. We test that it can be set (and it works).
2. If the default value changes in future the test will not become broken.

Reply:
changed